### DISCUSSION TITLE
rabbitmq.conf in 3.11.x supports environment variable interpolation

### DISCUSSION THREAD

**Discussion by michaelklishin** [docker-library/rabbitmq#627] [General]
@yosifkit @tianon FYI, [modern RabbitMQ versions support env variable interpolation](https://rabbitmq.com/configure.html#env-variable-interpolation) in `rabbitmq.conf`.

I've missed this feature but it's been in place in our config parser (Cuttlefish) since early 2021.

I would still not recommend using environment variables heavily where `rabbitmq.conf` or [definitions](https://rabbitmq.com/definitions.html) would do but it might
be useful to some image users, so I'm making sure that you are aware of this feature :)

**Comment by tianon** [comment]
Thanks for the heads up! :bow:
